### ISSUE TITLE
Split Beijing vs. London processing, handle custom feature engineering paths

### ISSUE THREAD

**Issue by LeoSalemann** [DS420-Cobras/DS420-Cobras#63]
NOTE: THIS IS ALL HAPPENING IN LEO'S BRANCHES
- [x] Separate London & Beijing paths for main.py and load_data.py

- [x]  customize data collection start date

- [x]  try including more fields

**Comment by LeoSalemann** [comment]
Tried all 6 algos with reduced data sets (Start in late April vs. beginning of mach ... all of the work

**Comment by ybhandari** [comment]
So, what is our strategy? Submit 1-2 median/smape submissions with march and 2-1 submissions with late April? How would we decide on the cutoff date for late April? Which algorithms are we going to use with which group?

**Comment by LeoSalemann** [comment]
I tested all 6 algo las night, and found moderate improvments for just about when I load_data() starting in late april instead of March 1. Smape, Median perfomed best, so I submitted those.  Also ran a regular fastPred, starting back in March.  Results are viewable in log.txt and OurSubmissions.csv.

Planning to maintain the same pattern, until more improvments become avaialble thru feature engineering.
